### PR TITLE
Final Manifest update

### PR DESCRIPTION
**Pull request by HarishH-DELL**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -4,15 +4,11 @@ annotations:
...
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.32.0
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
 
   # Annotations for testing.
   operators.operatorframework.io.test.mediatype.v1: scorecard+v1
   operators.operatorframework.io.test.config.v1: tests/scorecard/
-
-  # Annotations to specify supported OCP versions.
-  com.redhat.openshift.versions: v4.13-v4.14
```
Comment (panigs7) [inline bundle/metadata/annotations.yaml:18]: Revert the changes in the file

File: deploy/operator.yaml [hunk 2]
```diff
@@ -1314,7 +1246,7 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 512Mi
+            memory: 256Mi
```
Comment (panigs7) [inline deploy/operator.yaml:1249]: Revert this change

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -2929,42 +3479,5 @@ spec:
...
-    name: metrics-powerscale
-  - image: docker.io/dellemc/csm-metrics-powermax:v1.2.0
-    name: metrics-powermax
-  - image: docker.io/dellemc/csm-metrics-powerflex:v1.7.0
-    name: metrics-powerflex
-  - image: docker.io/dellemc/podmon:v1.8.0
-    name: podmon-node
-  - image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
-    name: kube-rbac-proxy
   replaces: dell-csm-operator.v1.3.0
```
Comment (panigs7) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:3482]: Revert the deleted lines from https://github.com/dell/csm-operator/pull/427/files#diff-684460b60b9e0858ebf0dbe4519848f01d8caef6b6499b8d5fe5b1d5b196913eL2932 to https://github.com/dell/csm-operator/pull/427/files#diff-684460b60b9e0858ebf0dbe4519848f01d8caef6b6499b8d5fe5b1d5b196913eL2968

**Review by panigs7 — APPROVED** [review]

**Review by chimanjain — APPROVED** [review]

File: bundle.Dockerfile [hunk 4]
```diff
@@ -22,4 +22,4 @@ LABEL com.redhat.delivery.operator.bundle=true
 # Copy files to locations specified by labels.
 COPY bundle/manifests /manifests/
 COPY bundle/metadata /metadata/
-COPY bundle/tests/scorecard /tests/scorecard/
+COPY bundle/tests/scorecard /tests/scorecard/
```
Comment (rajendraindukuri) [inline bundle.Dockerfile:25]: @HarishH-DELL Can you please revert this
